org.eclipse.wb.swing/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,8 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.wb.swing;singleton:=true
 Bundle-Version: 1.9.1.qualifier
-Bundle-ClassPath: .
+Bundle-ClassPath: .,
+ swing2swt.jar
```

Comment:
Since that library is not generated anymore, it should not be added to gitignore. Please remote the entry.